src/Model.ts

```diff
@@ -279,7 +280,7 @@ export default class Model extends Base {
     })
 
     const { promise, abort } = apiClient()[method](
-      this.url(),
+      endpoint || this.url(),
```

Comment:
API design question. Have you considered `${this.url()}endpoint`? I'm assuming, following rest conventions, that those are going to be nested. Example:

  - Base: `/tasks`
  - Specific domain: `/tasks/approve`

Reply:
Hard to answer. Both cases seem legit to me. What about checking first char of endpoint to know if we should do it relative or absolute?

Reply:
Another case. What if you want to update a resource or fetch a resource from different backend components?

Imagine. Component A: normal tasks. Component B: Tasks with specific logic for that backend component. 

maybe the routing in backend lives in different parts and is not nested

Reply:
then let's make it absolute. Is more flexible, no?

Reply:
Ok, it is absolute right now.